### ISSUE TITLE
The 'Cancel' button is not visible in Python 3.x

### ISSUE THREAD

**Issue by my8100** [scrapy/scrapyd#312]
It should be ```if b'cancel.json' not in self.root.children:``` for Python 3.x

https://github.com/scrapy/scrapyd/blob/d3f47f3d252373fb34d6fa99a93432909cf72fc9/scrapyd/website.py#L141-L143

**Comment by Digenis** [comment]
I think the real problem is that we read the configuration without regard to encoding.
https://github.com/scrapy/scrapyd/blob/d3f47f3d252373fb34d6fa99a93432909cf72fc9/scrapyd/config.py#L19-L23
Does this work for you?
```python
    self.cp = SafeConfigParser()
    self.cp.readfp(StringIO(default_config))
    self.cp.read(sources)
    for fname in extra_sources:
        with io.open(fname) as fp:
            self.cp.readfp(fp)
```

**Comment by my8100** [comment]
No, it still doesn't work for Python 3
https://github.com/scrapy/scrapyd/blob/d3f47f3d252373fb34d6fa99a93432909cf72fc9/scrapyd/website.py#L141
The result of `print(self.root.children.keys())`:
`dict_keys([b'', b'logs', b'jobs', b'schedule.json', b'cancel.json'...`

**Comment by my8100** [comment]
@Digenis Actually, I really doubt who would deactivate cancel.json.

**Comment by my8100** [comment]
@Digenis
The problem still exists in Python 3.7.
Did you test before merging #315?

**Comment by Digenis** [comment]
fixed in e3c9f91

**Comment by Digenis** [comment]
I only tested whether it reads correctly a file with non-ascii-encodable resource names.
But these were different bugs. Something that I completely forgot when I opened #315.
